backend/api/Migrations/20230830075517_RemoveInspectionFrequency.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Api.Migrations
+{
+    /// <inheritdoc />
+    public partial class RemoveInspectionFrequency : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "InspectionFrequency",
+                table: "MissionDefinitions");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "InstallationId",
+                table: "Plants",
+                type: "nvarchar(450)",
+                nullable: false,
```

Comment:
Why are these changes being done if they are already in migration [RequirePlantAndInstallation](https://github.com/equinor/flotilla/blob/main/backend/api/Migrations/20230811072130_RequirePlantAndInstallation.cs)?

Reply:
These are automatically generated so the underlying script must have detected that the change was not correctly in the model snaphsot until now. It is generally not a good idea to make manual changes here as this could mess up future auto-generated migrations, and it is difficult to correctly maintain the up/down functions as well as the snapshot.

Reply:
I will try to regenerate them though to see if that fixes it

Reply:
I can't see how removing this redundant code will break anything, so I just manually simplified the migration.